persistence-jdbc/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/persistence/jdbc/JdbcPersistenceComponents.scala

```diff
@@ -44,6 +44,8 @@ private[lagom] trait SlickProviderComponents extends DBComponents {
     )(executionContext)
     new SlickProvider(actorSystem, coordinatedShutdown)(executionContext)
   }
+  // We should eagerly init SlickProvider with Play 2.8.2+
+  slickProvider
```

Comment:
Really?

In production any usage should initialize it so I don't think there's a need to do it eagerly.

I think the changes in Play `2.8.2` affect tests mostly.

Reply:
> Really?

Nop 😞  
And it's the reason why I do not like this approach that fixes tests.

I can to add eager initialization in `com.lightbend.lagom.scaladsl.testkit.ServiceTest#startServer`, but for this solution `testkit-scaladsl` will depends on `persistence-jdbc-scaladsl` for compile scope.

Reply:
@ignasi35 Can be reviewed again 😄